src/routes/footage.route.ts

```diff
@@ -15,6 +16,8 @@ const footageRoute = (): Router => {
 
   router.get('/:id', getFootage);
 
+  router.get('/user/:id', getUserFootage);
+
   // router.get('/user/:id', getUserFootage);
```

Comment:
You've uncommented but kept the comment? :D

Reply:
So it turns out I was blind, I dmed b3kn about the endpoint url for getUserFootage until I realized he commented it in there and totally forgot to remove it LOL. I'll do it now.